build_defs.bzl

```diff
@@ -77,6 +77,7 @@ def pybind_extension(
             "@platforms//os:windows": name + ".pyd",
             "//conditions:default": name + ".so",
         }),
+        **kwargs
```

Comment:
For now, could we limit this to `testonly` and `visibility` as per line 71? (Bonus points if you add `visibility` above too!)

Reply:
Done

Reply:
Many thanks!